lib/node_modules/@stdlib/math/base/complex/divide/lib/cdiv.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+// MODULES //
+
+var FLOAT64_BIGGEST = require( '@stdlib/math/constants/float64-max' );
+var FLOAT64_SMALLEST = require( '@stdlib/math/constants/float64-smallest-normal' );
+var EPS = require( '@stdlib/math/constants/float64-eps' );
+var abs = require( '@stdlib/math/base/special/abs' );
+var max = require( '@stdlib/math/base/special/max' );
+var robustInternal = require( './robust-internal' );
+
+// VARIABLES //
+
+var LARGE_THRESHOLD = FLOAT64_BIGGEST * 0.5;
+var SMALL_THRESHOLD = FLOAT64_SMALLEST * ( 2.0 / EPS );
+var RECIP_EPS_SQR = 2.0 / ( EPS * EPS );
+
+// Create a two-element array in order to pass numbers locally
+// without allocating unnecessary tiny arrays.
+var buffer = [0, 0];
+
+// MAIN //
+
+/**
+* Divides two complex numbers.
+*
+* Based on the algorithm found in:
```

Comment:
We typically use Markdown in extended JSDoc comments. So, e.g., 

``` 
/**
* Divides two complex numbers.
*
* ## References
*
* * <reference-citation>
*
*
* @param {number} ...
* ...
*/
```

Next, for the reference, we use a tool to generate a Chicago-style citation

``` bash
$ make citation-reference CITATION=@baudin:2012
```